common/culture/innovations/wc_regional_innovations.txt

```diff
@@ -35,6 +49,26 @@ innovation_elephantry = {
 	flag = tribal_era_regional
 }
 
+innovation_dragonhawk = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	region = world_innovation_dragonhawks
+	icon = @dragonhawk
+
+	potential = {
+		##Not Taur, Flying Culture, or Giant
+		taur_culture_trigger = no
+		winged_culture_trigger = no 
+		is_giant_culture_trigger = no
+		NOT = {
+			this = culture:naga
+		}
```

Comment:
I think we need to create a scripted trigger. Call it something like... can ride a winged creature?